thunder.profile

```diff
@@ -26,22 +26,26 @@ function thunder_form_install_configure_form_alter(&$form, FormStateInterface $f
  * Implements hook_install_tasks().
  */
 function thunder_install_tasks(&$install_state) {
-
   $tasks = [
-    'thunder_module_configure_form' => [
-      'display_name' => t('Configure additional modules'),
-      'type' => 'form',
-      'function' => 'Drupal\thunder\Installer\Form\ModuleConfigureForm',
-    ],
-    'thunder_module_install' => [
-      'display_name' => t('Install additional modules'),
-      'type' => 'batch',
-    ],
     'thunder_finish_installation' => [
       'display_name' => t('Finish installation'),
     ],
   ];
 
+  if (empty($install_state['config_install_path'])) {
+    $tasks = [
+      'thunder_module_configure_form' => [
+        'display_name' => t('Configure additional modules'),
+        'type' => 'form',
+        'function' => 'Drupal\thunder\Installer\Form\ModuleConfigureForm',
+      ],
+      'thunder_module_install' => [
+        'display_name' => t('Install additional modules'),
+        'type' => 'batch',
+      ],
+    ] + $tasks;
+  }
+
```

Comment:
Something like this would be more compact:

`
function thunder_install_tasks(&$install_state) {
  if (empty($install_state['config_install_path'])) {
    $tasks['thunder_module_configure_form'] = [
      'display_name' => t('Configure additional modules'),
      'type' => 'form',
      'function' => 'Drupal\thunder\Installer\Form\ModuleConfigureForm',
    ];
    $tasks['thunder_module_install'] = [
      'display_name' => t('Install additional modules'),
      'type' => 'batch',
    ];
  }

  $tasks['thunder_finish_installation'] = [
    'display_name' => t('Finish installation'),
  ];

  return $tasks;
}
`